starlight/fixtures/fixtures.json

```diff
@@ -0,0 +1,711 @@
+[
```

Comment:
The fixtures here should only include data from our Django app `starlight`. Check out the `--help` for the `dumpdata` command if you need guidance.

Reply:
I think its properly filtered now, but Github still isnt marking this as resolved